wordpress-mobile/WordPress-Android
# Reader Detail Updates (Expandable Tags View)

Task: #12900

This PR 
- adds expandable tags view to the reader post details header view
- removes existing tags strip with popular tags from the  reader post details content layout

Light Mode | Dark Mode
------------|-------------
![expandable tags view](https://user-images.githubusercontent.com/1405144/94914731-363f7c80-04c9-11eb-9c25-072bd7a6f089.png) | ![expandable tags view dark](https://user-images.githubusercontent.com/1405144/94914764-46eff280-04c9-11eb-95de-df60e024f790.png)

To test:

- Launch the app and go to the Reader tab
- Tap on a card with some tags
- Notice that expandable tags view is visible on the details screen and works properly

PR submission checklist:
